orttraining/orttraining/eager/test/test_argmax.py

```diff
@@ -0,0 +1,39 @@
+import onnxruntime_pybind11_state as torch_ort
+import torch
+
+device = torch_ort.device()
+cpu_tensor = torch.rand(3, 5, 7)
+
+cpu_out_tensor = torch.tensor([])
+cpu_result = torch.argmax(cpu_tensor, out=cpu_out_tensor)
```

Comment:
builds failing:
Traceback (most recent call last):
  File "/build/Release/eager_test/test_argmax.py", line 8, in <module>
    cpu_result = torch.argmax(cpu_tensor, out=cpu_out_tensor)
RuntimeError: Expected out tensor to have dtype long int, but got float instead

Reply:
This file was not meant to be included - removed!